strapi/strapi
# [DEITS] add the cli backbone

### What does it do?

- Adds the export command (works in theory, but transfer engine isn't implemented yet)
- Adds the import command (no idea if it works, but the backbone is there and it builds)
 
### Why is it needed?

So the export code can be run from the command line

### How to test it?

Run `yarn strapi export --help` and you will see help for the command.
Run `yarn strapi export` and you will get an error that no file was created (because the export code isn't fully implemented yet)

### Related issue(s)/PR(s)
